tests/mock_boto3/case.py

```diff
@@ -16,6 +16,7 @@
 import itertools
 import os
 import shutil
+import stat
```

Comment:
Missed this earlier. Where are we using this?

Reply:
whoops, we're not. you can delete it (or remove it in pre-release pyflaking)